src/classes/CallableApiParameters.cls

```diff
@@ -49,6 +49,9 @@ public class CallableApiParameters {
     public static final String PARAM_PROCESSING_BOUNDARY = 'ProcessingBoundary';
     public static final String PARAM_DISABLE_ALLOCATION_TRIGGERS = 'DisableAllocationTriggers';
     public static final String PARAM_SCHEDULE_JOBS = 'ScheduleJobs';
+    public static final String PARAM_SCHEDULED_JOB_NAME = 'ScheduledJobName';
+    public static final String PARAM_SCHEDULED_JOB_CRON_EXPRESSION = 'ScheduledJobCronExpression';
+    public static final String PARAM_SCHEDULED_JOB_CLASS = 'ScheduledJobClass';
```

Comment:
I think we can remove the "ScheduledJob" part of the param values to since the params are only used as part of the job.schedule task:
```suggestion
    public static final String PARAM_SCHEDULED_JOB_NAME = 'JobName';
    public static final String PARAM_SCHEDULED_JOB_CRON_EXPRESSION = 'CronExpression';
    public static final String PARAM_SCHEDULED_JOB_CLASS = 'ApexClass';
```